### PR TITLE
add basic material-ui example

### PR DESCRIPTION
**Pull request by sgiangola**
Adding a basic example using material-ui's styling

### CODE REVIEW

File: examples/material-ui/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app) and Rescripts.
+
+You can:
+
+- [Open this example in a new CodeSandbox](https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/basic)
```
Comment (MareoRaft) [inline examples/material-ui/README.md]: shouldn't this url be 

https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/material-ui

or similar?  Have you created a codesandbox for your example?

File: examples/material-ui/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "scripts": {
+    "start": "rescripts start",
+    "build": "rescripts build",
+    "test": "rescripts test",
+    "eject": "rescripts eject"
+  },
+  "dependencies": {
+    "@material-ui/core": "^4.4.3",
+    "eslint-config-prettier": "^6.3.0",
+    "material-ui": "^0.20.2",
```
Comment (MareoRaft) [inline examples/material-ui/package.json]: Is this package "material-ui" in use?  Perhaps this was installed by mistake the first time you tried to install Material UI.

**Comment by MareoRaft** [comment]
@sgiangola Thanks for that change.  The link https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/material-ui is dead though.  To be clear, I was asking if you could create a working codesandbox demo that demos your example.  This would demonstrate (1) that your example works and (2) what your theme looks like.  It would allow people to quickly determine if they want to use your theme or not.

Thanks and looking forward to using it myself!

(note: I am not a react-table contributor myself.  I am just a developer interested in using material-ui theming with react-table, so i've been looking into your example.)

**Comment by tannerlinsley** [comment]
So far I haven't been able to see the example yet. I would expect this codesandbox import link to work:

https://codesandbox.io/s/github/sgiangola/react-table/tree/material-ui/examples/material-ui

Please create a temporary codesandbox that we can see, then we can merge the PR and change the link to:

https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/material-ui

File: examples/material-ui/package.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  "private": true,
+  "scripts": {
+    "start": "rescripts start",
+    "build": "rescripts build",
+    "test": "rescripts test",
+    "eject": "rescripts eject"
+  },
+  "dependencies": {
+    "@material-ui/core": "^4.4.3",
+    "eslint-config-prettier": "^6.3.0",
```
Comment (Julian-B90) [inline examples/material-ui/package.json]: eslint-config-prettier ist not an `dependencies` it is an `devDependencies`

**Comment by tannerlinsley** [comment]
It looks like #1557 was ready to go, so we went ahead and merged it. That should cover the same use case here, so I'm going to close this PR unless there are any drastic differences in the implementation of #1557 that warrant a whole new example. Thanks guys!

**Pull request closed without merging**